packages/libs/react-components/src/components/Card/styles.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+
+import { styled } from '../../styles';
+
+export const CardColors: string[] = [
+  'default',
+  'primary',
+  'secondary',
+  'positive',
+  'warning',
+  'negative',
+];
+
+export const StyledCard = styled('div', {
+  boxSizing: 'border-box',
+  backgroundColor: '$colors$neutral1',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  padding: '24px',
+  minWidth: '546px',
+  width: 'min-content',
+  height: 'min-content',
+  boxShadow: '0px 0px 10px rgba(0, 0, 0, 0.04)',
+  borderRadius: '8px',
+  variants: {
+    color: {
```

Comment:
@isaozler Should the card component have so many color options? I would expect it to just be gray

Reply:
We have 2 colors for card surfaces, white and gray indeed